yarn-project/pxe/src/pxe_oracle_interface/pxe_oracle_interface.ts

```diff
@@ -490,20 +490,14 @@ export class PXEOracleInterface implements ExecutionDataProvider {
         // a new set of secrets and windows to fetch logs for.
         const newLargestIndexMapForIteration: { [k: string]: number } = {};
 
-        // Fetch the logs for the tags and iterate over them
-        const logsByTags = await this.aztecNode.getLogsByTags(tagsForTheWholeWindow);
+        // Fetch the private logs for the tags and iterate over them
+        const logsByTags = await this.aztecNode.getPrivateLogsByTags(tagsForTheWholeWindow);
```

Comment:
Here we could call one of the two internal helpers that end up doing `node.getLogsByTags`, and so not change the node interface (yet).

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/14459/commits/2d318b363423dadc6cd342be1de104695706c6ae